superset/assets/visualizations/nvd3_vis.js

```diff
@@ -18,6 +18,7 @@ import './nvd3_vis.css';
 import { VIZ_TYPES } from './main';
 
 const minBarWidth = 15;
+const minMarginPad = 50;
```

Comment:
this should probably be called `maxMarginPad` and have a little comment of what it does.